framework/sandstone.cpp

```diff
@@ -660,6 +660,37 @@ inline void test_the_test_data<true>::test_tests_finish(const struct test *the_t
 #undef maybe_log_error
 }
 
+uint64_t sandstone_user_cpu_time(uint64_t from)
```

Comment:
`sandstone_test_utils.h` feels like a better home for these two functions (`sandstone_user_cpu_time` and `sandstone_sys_cpu_time`). Also, please add unit tests for these.